plugins/janus_videocall.c

```diff
@@ -786,7 +786,7 @@ void janus_videocall_incoming_rtp(janus_plugin_session *handle, janus_plugin_rtp
 				json_t *result = json_object();
 				json_object_set_new(result, "event", json_string("simulcast"));
 				json_object_set_new(result, "videocodec", json_string(janus_videocodec_name(session->vcodec)));
-				json_object_set_new(result, "substream", json_integer(session->sim_context.substream));
+				json_object_set_new(result, "substream", json_integer(peer->sim_context.substream));
```

Comment:
This is wrong. The substream returned here is the substream the user (so `session`) wants from the peer. Substream and temporal info are always receiver related.

Reply:
disagree. 
 I think peer->sim_context.substream is right.   you can check it seeing the substream status on videocall webpage.  I have verified it.   Using peer, the showing is ok.  if using session, the status showing is abnormal.
Peer is the callee.   The substream returned here is what substream the callee wants. It is showed on webpage.

Reply:
Ah you're rght, sorry: this is an event we're sending to `peer`, not `session` as I thought :+1:   The `session->vcodec` above threw me off.